polling_stations/apps/councils/models.py

```diff
@@ -97,3 +99,8 @@ class CouncilGeography(models.Model):
     )
     gss = models.CharField(blank=True, max_length=20)
     geography = models.MultiPolygonField(null=True)
+
+
+class UserCouncils(models.Model):
+    user = models.ForeignKey("auth.User", on_delete=models.CASCADE)
+    council = models.ForeignKey(Council, on_delete=models.CASCADE)
```

Comment:
Yep, I'd considered that this would be a problem but not really thought about it.

One thing to think about though: the way we're doing a (form of) `get_or_create` on user accounts (using the OTT) means that it doesn't actually matter if a User is `CASCADE DELETE`d at some point: it'll just get re-created next time they log in.

Also, as council records change infrequently (their existence at least), and when they do it''s normally not around the time that we'll be getting data, it seems reasonable to run the `import_councils` command on the logger DB and just use that for the JOIN to users.

Of course, there's bound to be some problems some of the time, but see the other comment about being designed to reduce work load rather than eliminate it for us.

We're also looking at moving to an _entirely different_ deployment model that I'd be interested in talking to you about at some point (the same as the WCIVF one, if you've been following that at all)

Reply:
> We're also looking at moving to an entirely different deployment model that I'd be interested in talking to you about at some point (the same as the WCIVF one, if you've been following that at all)

To an extent.. I'd be interested in hearing about it. Feel free to give me a shout.